xivModdingFramework/Textures/FileTypes/Tex.cs

```diff
@@ -619,14 +619,14 @@ private static async Task<byte[]> SwapRBColors(byte[] textureData, int width, in
                         {
                             for (var x = 0; x < width; x++)
                             {
-                                var red = br.ReadByte();
-                                var green = br.ReadByte();
                                 var blue = br.ReadByte();
+                                var green = br.ReadByte();
+                                var red = br.ReadByte();
                                 var alpha = br.ReadByte();
 
-                                convertedBytes.Add(blue);
-                                convertedBytes.Add(green);
                                 convertedBytes.Add(red);
+                                convertedBytes.Add(green);
+                                convertedBytes.Add(blue);
                                 convertedBytes.Add(alpha);
```

Comment:
At first I thought the issue occurred here because we were reading the channels in the wrong order, but I later realised we also added them back incorrectly which cancelled out... Nothing changed here functionally other than that I fixed the var names.